cartographer/mapping_3d/sparse_pose_graph.cc

```diff
@@ -424,6 +424,16 @@ SparsePoseGraph::GetTrajectoryNodes() {
   return trajectory_nodes_.data();
 }
 
+std::shared_ptr<const Submap> SparsePoseGraph::GetSubmap(
+    const mapping::SubmapId& submap_id) {
+  common::MutexLocker locker(&mutex_);
+  // Conceptually, this method should only be used to ask for finished submaps.
```

Comment:
Add a comment to the struct definition that `state` is about the state of the submap in the background processing?